code/datums/components/customizable_reagent_holder.dm

```diff
@@ -123,6 +123,11 @@
 		return
 	if (replacement)
 		var/atom/replacement_parent = new replacement(atom_parent.loc)
+		if(atom_parent.loc == attacker && isitem(replacement_parent) && isitem(atom_parent))
+			var/obj/item/I = atom_parent
+			if(attacker.is_holding(I))
+				var/obj/item/R = replacement_parent
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names